dali/test/python/jax/test_integration.py

```diff
@@ -47,7 +47,7 @@ def dali_pipeline():
 
```

Comment:
A comment to the line above (I cannot do it in this review): direct use of fn.constant is not recommended. Use `dali.types.Constant` instead.